### ISSUE TITLE
Feature Request: official event for pausing and starting preloading 

### ISSUE THREAD

**Issue by szymonhernik** [muxinc/elements#853]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Mux Elements/Packages does this apply to? Select all that apply

mux-player, mux-player-react

### Description

I wish there was an option that stops preloading/buffering after you start the video. This would be useful in cases like when the video leaves the screen stop fetching the video data. Currently, I found a walkaround this by doing `_hls.stopLoad()` and `_hls.stopLoad()` on the player but I don't think this is official solution. 

This has been discussed here as well:
https://github.com/muxinc/elements/discussions/422

### Expected Behavior

Stop and start prerendering video data after calling a function.

**Comment by mmcc** [comment]
Yeah, agreed, having to dig into the underlying playback tech directly to do this doesn't feel right. Thanks for the feedback, let us think on what the path could look like here.

**Comment by spuppo-mux** [comment]
Hello, thanks for raising this. The current workaround is fine to use for now; we are going to take this into account for future versions. You may also want to look into `Hls.pauseBuffering()` and `Hls.resumeBuffering()` ([reference](https://hlsjs.video-dev.org/api-docs/hls.js.hls)). I do want to note that these functions behave a bit differently on Safari and Chrome with `mse` playback.
